The advent of high throughput DNA sequencing has enabled the unprecedented  ability to read the 3 billion As, Cs, Gs, and Ts that make up our genome. This has led to the discovery of thousands of mutations that are  associated with different human diseases. The vast majority of these associations, however, remain exactly that - a single change in our DNA that is statistically associated with a disease. 
%

This dissertation describes three studies that aim to understand transcriptional regulatory logic and leverage this understanding to place human mutations in the specific  biological pathways and developmental contexts that give rise to different diseases. In the first study, I investigated the evolution of gene regulation in the developing neocortex driven by transposable elements being co-opted into developmental enhancers. In the second study, I examined genes that were mutated in autism spectrum disorders, identified a single protein that regulates the majority of them during a specific period of brain developmental, and exploited this property to confirm known autism genes and prioritize others for further study. Finally, I developed a high-specificity method to map non-protein-coding disease mutations from humans to homologous regions in the DNA of other species, where we re-created the human disease using CRISPR/Cas9 genome editing and studied the effects of the mutation. These studies demonstrate how genomics can extend our understanding of different diseases by offering strategies for revealing their precise mechanism of action.
%

The neocortex is a mammalian-specific structure that is responsible for higher functions such as cognition, emotion, and perception. To gain insight into its evolution and the gene regulatory codes that pattern it, we studied the overlap of its active developmental enhancers with transposable element families and compared this overlap to uniformly shuffled enhancers. Here we show a striking enrichment of the MER130 repeat family among active enhancers in the mouse dorsal cerebral wall, which gives rise to the neocortex, at embryonic day 14.5 (E14.5). We show that MER130 instances preserve a common code of transcriptional regulatory logic, function as enhancers, and are adjacent to critical neocortical genes. MER130, a nonautonomous interspersed transposable element, originates in the tetrapod or possibly Sarcopterygii ancestor, which far predates the appearance of the neocortex. Our results show that MER130 elements were recruited, likely through their common regulatory logic, as neocortical enhancers.
%

Exome sequencing studies have identified multiple genes harboring \emph{de novo} loss-of-function (LoF) variants in individuals with autism spectrum disorders (ASD), including \emph{TBR1}, a master regulator of cortical development. We performed ChIP-seq for TBR1 during mouse cortical neurogenesis and show that TBR1-bound regions are enriched adjacent to ASD genes. ASD genes were also enriched among genes that are differentially expressed in \emph{Tbr1} knockouts, which together with the ChIP-seq data, suggests direct transcriptional regulation. Of the 9 ASD genes examined, 7 were misexpressed in the cortices of \emph{Tbr1} knockout mice, including 6 with increased expression in the deep cortical layers. ASD genes with adjacent cortical TBR1 ChIP-seq peaks also showed unusually low levels of LoF mutations in a reference human population and among Icelanders. We then leveraged TBR1 binding to identify an appealing subset of candidate ASD genes. Our findings highlight a TBR1-regulated network of ASD genes in the developing neocortex that are relatively intolerant to LoF mutations, indicating that these genes may play critical roles in normal cortical development.
%

More than 90\% of human disease-associated mutations lie in non-genic regions of the genome, presenting a challenge for interpretation. We report a screen to identify GWAS SNPs embedded in deeply conserved non-coding DNA elements (CNE) preserved syntenically across vertebrates. We found 22 CNE/SNP pairs covering a wide range of human diseases and validated a subset for enhancer activity and SNP regulatory function, including CNE1/SNP rs17421627 associated with retinal vasculature defects. Zebrafish CNE1:EGFP transgenics revealed human CNE1 enhancer activity in the retina. CRISPR/Cas-9 deletion of CNE1 in the zebrafish genome led to defects in blood vessel development in the retina, suggesting that CNE1 enhancer is functionally associated with retinal vascular caliber in humans. Introducing the risk mutation in CNE1 abolished EGFP expression, indicating that SNP rs17421627 is likely the causal mutation responsible for the human phenotype. Validating our prediction, we identified the target gene \emph{cis}-regulated by CNE1/rs17421627: the neurogenesis modulator, microRNA-9. Consistent with the GWAS studies, miR-9 depletion also led to retinal vasculature defects, demonstrating that \emph{miR-9} is the conserved gene functionally associated with retinal blood vessel development. This study validates our \emph{in silico} approach to identify conserved enhancers regulated by GWAS SNPs and demonstrates how the combination of human and zebrafish genetics reveals enhancer function, the regulatory activity of the SNP, the \emph{cis}-regulated gene, and the biological processes disrupted in the associated disorders.
%

These studies demonstrate how genomics can extend our understanding of different diseases by offering strategies for combining human genetics and developmental biology to reveal their precise mechanism of action.
