\chapter{Supplementary material for \chapref{chap:great}}
\label{chap:greatSuppl}

\section{Ontologies supported}
\label{XOntologies}
GREAT assimilates knowledge from 20 separate ontologies containing biological
knowledge about gene functions, phenotype and disease associations,
biological pathways, gene expression data, presence of regulatory motifs, and
gene families (\tabref{tab:supplOntosFull}).
Statistics for each ontology list the total number of terms in the ontology
that are currently tested by GREAT, the number of genes
annotated with one or more terms in the ontology, and the number of direct
associations between ontology terms and genes (\tables{tab:supplOntoHuman}
and~\ref{tab:supplOntoMouse}).  Some ontologies contain parent/child
relationships between terms expressed as a directed acyclic graph; general
terms within these ontologies inherit genes that are only labeled with more
specific child terms as indirect associations.
To increase statistical power (by reducing the multiple hypothesis correction factor),
GREAT does not test any general term whose associated gene list is
identical to the associated gene list of a more specific child term.  The
following ontologies are currently used:

\subsection{Gene Ontology}
The \emph{Gene Ontology} (GO; \url{http://www.geneontology.org/}) provides a controlled vocabulary to describe attributes of
gene products~\citep{Ashburner2000}.  GO contains three separate ontologies that describe molecular functions, biological
processes, and cellular components of proteins.

\subsection{Mouse Phenotype}
The Mouse Genome Informatics (MGI) resource contains data about mouse
genotype--phenotype associations primarily obtained via
literature curation~\citep{Blake2009,Bult2008}
(\url{http://www.informatics.jax.org/phenotypes.shtml}). Phenotypic terms are
canonicalized and relationships between terms are enumerated in the Mammalian
Phenotype Ontology~\citep{Smith2005}.

\subsection{MSigDB Ontologies}
The Molecular Signatures Database (MSigDB; \url{http://www.broad.mit.edu/gsea/msigdb/}) contains a collection of
gene sets~\citep{Subramanian2005}. 
The following description of the various ontologies within MSigDB is taken from
\url{http://www.broad.mit.edu/gsea/msigdb/collections.jsp}.

\begin{itemize}
\item \emph{MSigDB Cancer Neighborhood}\\
Computational gene sets defined by mining large collections of
cancer-oriented microarray data. Gene sets defined by expression
neighborhoods centered on 380 cancer-associated
genes~\citep{Brentani2003}. This collection is identical to that previously
reported in~\citep{Subramanian2005}.
\item \emph{MSigDB Cancer Modules}\\
Computational gene sets defined by mining large collections of
cancer-oriented microarray data~\citep{Segal2004}.  Briefly, the authors
compiled gene sets (``modules'') from a variety of resources such as KEGG,
GO, and others. By mining a large compendium of cancer-related microarray
data, they identified 456 such modules as significantly changed in a variety
of cancer conditions.
\item \emph{MSigDB Pathway}\\
Gene sets from pathway databases. Usually, these gene sets are canonical
representations of a biological process compiled by domain experts.
\item \emph{MSigDB Perturbation}\\
Gene sets that represent gene expression signatures of genetic and chemical perturbations. 
\item \emph{MSigDB Predicted Promoter Motifs}\\
Sets of genes that share a transcription factor binding site defined in the TRANSFAC (version 7.4,
\url{http://www.gene-regulation.com/}) database. Each of these gene sets is annotated by a TRANSFAC record.
\item \emph{MSigDB miRNA Motifs}\\
Sets of genes that share a 3'-UTR microRNA binding motif.
\end{itemize}

\subsection{PANTHER Pathway}
\emph{PANTHER Pathway} (\url{http://www.pantherdb.org/pathway/}) contains information on biological 
pathways (primarily signaling pathways)~\citep{Mi2007}. PANTHER pathways are
collections of biological molecules and the reactions in which they
participate. Only well-documented reactions and relationships are listed.

\subsection{Pathway Commons}
\emph{Pathway Commons}~\citep{Cerami2006} contains a comprehensive collection of pathways from multiple sources listed at 
\url{http://www.pathwaycommons.org/pc/}.  According to the website, ``[p]athways include biochemical reactions, complex
assembly, transport and catalysis events, and physical interactions involving proteins, DNA, RNA, small molecules and
complexes.''

\subsection{BioCyc Pathway}
\emph{BioCyc} (\url{http://biocyc.org/}) contains information linking genes to the metabolic pathways in which they
participate~\citep{Caspi2008}.

\subsection{MGI Expression:\ Detected \& Not Detected}
The Gene Expression Database (GXD,
\url{http://www.informatics.jax.org/expression.shtml}), a part of the Mouse
Genome Informatics database, contains expression data with a focus on gene
expression during mouse development~\citep{Smith2007,Bult2008}.  The
information is primarily obtained from the literature via manual
curation. Each entry gives the expression in a specific anatomical structure
during a specific developmental period or ``Theiler
stage''~\citep{Theiler1989}.  The anatomy for each developmental stage is
represented by a directed acyclic graph that gives a hierarchy of anatomical
terms and their relationships. The database contains information about which
genes are expressed and which are not found to be expressed.

We represent the MGI Gene Expression Database by several sub-ontologies,
where each sub-ontology is specific to a developmental stage. We then combine
all sub-ontologies into one ontology so that all developmental stages are
tested at once.  \emph{MGI Expression:\ Detected} contains data about genes
that are expressed and \emph{MGI Expression:\ Not Detected} contains data
about genes whose expression is measured but not experimentally detected.

The human \emph{MGI Expression} ontologies are derived by mapping expression
information from all genes in the mouse ontologies to their human orthologs
and assume large-scale conservation of expression patterns.

\subsection{Transcription Factor Targets}
The \emph{Transcription Factor Targets} ontology contains transcription factor (TF) target sets for human and mouse collected from literature~\citep{Linhart2008}. 
Most TF target genes were identified by ChIP-chip experiments (see \url{http://acgt.cs.tau.ac.il/amadeus/suppl/metazoan_compendium.htm}).

\subsection{miRNA Targets}
The \emph{miRNA Targets} ontology contains miRNA target sets for human and mouse collected from literature~\citep{Linhart2008}. 
miRNA target genes were identified as genes downregulated after miRNA overexpression (see \url{http://acgt.cs.tau.ac.il/amadeus/suppl/metazoan_compendium.htm}).

\subsection{InterPro}
\emph{InterPro} (\url{http://www.ebi.ac.uk/interpro/}) is a database of protein domains, families and functional
sites~\citep{Hunter2009}. 
InterPro annotations give information about the function, structure and evolution of the domains. 
InterPro combines data from several other databases (PROSITE, PRINTS, Pfam, ProDom, SMART, TIGRFAMs, PIRSF, SUPERFAMILY,
PANTHER and Gene3D).

\subsection{TreeFam}
The Tree families database (TreeFam, \url{http://www.treefam.org/}) contains information about the evolutionary history
(both orthologs and paralogs) of gene families~\citep{Ruan2008}. 
A gene family is defined as ``a group of genes that evolved after the speciation of single-metazoan animals''.
We format this data into an ontology by creating an ontology term for each gene family and then associating
each gene within the species (human or mouse) with its gene family.

\subsection{HGNC Gene Families}
The HUGO Gene Nomenclature Committee groups genes into gene families based on sequence similarity, data from the literature
and other databases, and manual curation~\citep{Bruford2008}.  The groupings are listed at
\url{http://www.genenames.org/genefamily.html}.

\subsection*{Website architecture}
\label{XArchitecture}
The GREAT website output is generated by way of server-side PHP code invoking
a Python wrapper over a C program.  The C code makes use of the UCSC Genome
Browser~\citep{Kent2002b} code libraries and performs the core calculations.
Test results are formatted for web display using Python and PHP code.  GREAT
uses the DataTable control from the Yahoo!\ User Interface Library
(\url{http://developer.yahoo.com/yui/}) and custom JavaScript code to allow
many user operations without need for more server data.  This allows
rapid responses for all filtering requests once the initial results
page has loaded locally.  Operations which generate new pages, such as
getting details for a single ontology term or the generation of publication
quality table display do involve return trips to the server, which are
handled by PHP code.

\subsection*{Graphical User Interface}
\label{XGUI}
The graphical user interface (GUI) of GREAT \greatversion~has the following components: 
\begin{itemize}
\item \emph{User Input} \\
The user input page (\supfig{fig:supplWorkflow}a) requires two inputs from
the user: the organism genome assembly in which the analysis should be
performed and the \cis-regulatory regions to analyze in BED
format~\citep{Kent2002b}.
Optionally, the user can upload a background set of
genomic regions to test against rather than the whole genome.
The user can also optionally alter the association
rules between \cis-regulatory regions and their putative target genes from the default
\emph{basal plus extension} rule via the Advanced options tab.

\item \emph{Global Output \& Controls} \\
Upon submission of a dataset, users are directed to the global output screen
of GREAT (\supfig{fig:supplWorkflow}b).  By default, only ontology terms
significant by both the binomial and hypergeometric tests using the multiple
hypothesis correction false discovery rate (FDR)~\citep{Benjamini1995} $\leq 0.05$ whose binomial fold enrichment is at least
2.0 are displayed.  The extensive global controls at the top of the page
allow users to change the ontologies shown, alter the data columns displayed
for each result term, and change the multiple test correction type and
threshold.  The number of enriched terms shown for each ontology, the display
of terms not significant by one or both of the enrichment tests, and the
filtering of terms by their descriptions can all be changed via the global
controls.

Within each ontology table, terms can be sorted by any data column except
FDR.  The data for a single table can be downloaded either as HTML or as a
tab-separated file.  By clicking on a particular term, additional information
is presented in an individual term page (described below).

\item \emph{Individual Term Page} \\
Each ontology term tested for enrichment has an associated individual term
page (\supfig{fig:supplWorkflow}c).  The page lists all \cis-regulatory
regions that reside within the regulatory domain of any gene annotated with
the term, all the genes annotated with the term that possess one or more
input regions within its regulatory domain, and the definition of the
ontology term from the source website inset into the page.  By clicking on
any \cis-regulatory region listed, users can navigate to the UCSC Genome
Browser with custom tracks of the entire input set and only the elements
contributing to the specific term enrichment (\supfig{fig:supplWorkflow}d).

\item \emph{Online Documentation} \\
User help documentation is available at \url{http://great.stanford.edu/help} and includes information regarding all
aspects of GREAT.

\item \emph{Demo Sets}\\
The SRF~\citep{Valouev2008} and limb p300 (\citep{Visel2009}) datasets presented in
the main text, as well as several additional sets, are available as
demonstration input sets for GREAT.  These sets can be tested by navigating
to the ``Demo'' link.

\end{itemize}

% The output display of GREAT for a whole genome test joins the results of both
% the binomial and hypergeometric enrichment tests (described above) for each
% term.  The ``summary'' view option, enabled by default, shows only ontology
% terms that pass the desired statistical significance threshold measured for
% both the binomial and hypergeometric tests.  The ``full'' view option can be
% used to see all tested ontology terms, including those not significant by
% either the binomial or hypergeometric tests.


% \subsection*{Demonstration sets}
% \label{XGREATDemos}
% The SRF~\citep{Valouev2008} and p300~\citep{Visel2009} datasets presented in
% the main text, as well as the set of ultraconserved
% elements~\citep{Bejerano2004}, are available as demonstration input sets for
% GREAT.  These sets can be tested by navigating to the ``Demo'' link from
% \greaturl.

\section{Additional ChIP-Seq analyses}
\label{XAddlChIPSeq} % index anchor 
To assess the ability of GREAT to improve upon existing gene-based analyses of ChIP-Seq datasets,
we compared GREAT enrichments to gene-based tool enrichments for multiple datasets.
%
Where they were not performed by the original authors,
we performed gene-based enrichment analyses using the Database for Annotation,
Visualization, and Integrated Discovery (DAVID)~\citep{Huang2007}.  
%
The analysis of next-generation binding data is not clearly mappable to
more advanced techniques like the Gene Set Enrichment Analysis (GSEA)~\citep{Subramanian2005}
which rank genes by the intensity difference of the different probes/genes and are
typically applied to compare gene expression profiles from two classes (e.g.\ people
with a disease vs.\ healthy controls).

For each ChIP-Seq dataset we analyzed using DAVID, we mapped the identified peaks that reside
within 2 kb of the TSS of the nearest gene as identified by the UCSC Known Genes
track~\citep{Hsu2006} to the nearby gene and ran enrichments over the resulting gene list.
%
The ten most enriched terms from each annotation cluster reported significant by DAVID at a
threshold of $0.05$ after an FDR multiple hypothesis correction are shown in each enrichment table.
%
To run a ``gene-based GREAT'', we used the \emph{basal plus extension} association rule with
basal upstream and downstream parameters both set to 2 kb and an extension of 0 bp and excluded the set
of curated regulatory domains.
The ten most enriched terms significant by the hypergeometric test at a threshold of $0.05$ after an
FDR multiple hypothesis correction are shown in each enrichment table.
%
Most \emph{Gene Ontology} enrichments are nearly identical between DAVID and ``gene-based GREAT'',
though some slight discrepancies occur due to different versions of GO and gene sets used.
%

%
Since GREAT performs a \cis-regulatory element-based test, no mappings from ChIP-Seq peaks
to genes are required for preprocessing.
%
For each ChIP-Seq dataset we analyzed using GREAT, we ran the identified peaks through GREAT
using its default settings (the \emph{basal plus extension}
association rule with basal domains extending 5 kb upstream and 1 kb downstream of the TSS
and extension to the basal domains of the nearest genes within 1 Mb).
%
The top ten terms significant at a threshold of $0.05$ by the binomial test after an FDR
multiple hypothesis correction that have a fold enrichment
of at least two and are also significant by the hypergeometric test
are shown in each enrichment table.
%

\subsection{P300 in mouse embryonic stem cells}
A recent ChIP-Seq analysis of transcription factors involved in the
maintenance of the self-renewal and pluripotency capabilities of embryonic
stem cells (ESCs) assayed genome-wide binding of p300 within mouse
embryonic stem cells (mESCs)~\citep{Chen2008}.
%
The binding profile of p300 was noted to
co-localize with the binding profiles of Nanog, Oct4, and
Sox2 (\citep{Chen2008}), which are known to be involved in stem cell
maintenance~\citep{Ivanova2006}.

When we ran DAVID on the associated gene set, no annotation terms were found
to be statistically significant (\tabref{tab:supplP300ESAll}a).  We
then analyzed all 524 identified p300 ChIP-Seq peaks~\citep{Chen2008} with
GREAT using the default settings (5+1 kb basal, up to 1 Mb extension).  GO enrichments
indicate that chromatin binding and transcriptional regulator proteins are
targets of p300 in ESCs, with striking enrichments for genes involved in stem
cell maintenance and stem cell differentiation
(\tabref{tab:supplP300ESAll}b).  The
\emph{MGI Expression:\ Detected} ontology shows enrichment for genes expressed
during the very first stages of development~\citep{Theiler1989}, consistent
with stem cell maintenance.  The \emph{Predicted Promoter Motifs} ontology
shows enrichment for p300 binding near genes whose promoters contain binding
sites for GTF3A and NHLH1.  GTF3A, which helps to assemble active chromatin,
is required for transcription of the 5S RNA genes that drive growth in early
developing embryos~\citep{Drew1995}.  While the significance of NHLH1 binding
to stem cell maintenance is not known, NHLH1 is known to be expressed in the
developing nervous system~\citep{Begley1992}.
%

The ``gene-based GREAT'' results identify stem cell differentiation as an enriched
term, but none of the other top enriched terms overlap the enrichments displayed
by GREAT (\tabref{tab:supplP300ESGBG}).
%
Running GREAT with a more limited extension (5+1 kb basal, up to 50 kb extension)
highlights more general terms as enriched and no longer emphasizes enrichment 
for genes expressed in early development (\tabref{tab:supplP300ESBasal50}).
%
Distal binding appears to contribute greatly to the function of p300 in embryonic
stem cells, as demonstrated by nearly 40\% of all binding occurring outside of
50 kb from the TSS of any gene (\figref{fig:gFig2}a).
%
Variation in distal association rules leads to generally similar enrichments as the
default GREAT, emphasizing genes involved in stem cell maintenance and expressed in
early development (\tables{tab:supplP300ESTwo}, \ref{tab:supplP300ESOne}).
%
% The dramatic difference between GREAT and DAVID results
% highlights the importance of distal binding events to gene regulation in mammalian species and the benefit
% of integrating distal binding events appropriately into an enrichment-testing framework.
%
% Note: stem cell differentiation is not bad:
% As defined by GO, stem cell differentiation is ``the process whereby a relatively unspecialized cell acquires
% specialized features of a stem cell.''
% 

\subsection{Signal transducer and activator of transcription 3 (Stat3) in mouse embryonic stem cells}
The binding events of Signal transducer and activator of transcription 3
(Stat3) were also assayed within mESCs using ChIP-Seq in
the study mentioned above~\citep{Chen2008}.
%
Stat3 is a transcriptional
activator whose activity is sufficient to maintain an undifferentiated state
of mouse ESCs~\citep{Matsuda1999}, but whose constitutive
expression has also been linked to various cancers~\citep{Bromberg1999}.
Stat3 transduces signals from the IL-6 family of cytokine receptors~\citep{Hirano2000}.
One of these cytokines, Leukemia Inhibitory Factor (LIF), is a component of media used
to culture ESCs in an undifferentiated state.
 
Gene-based DAVID enrichments for the Stat3 dataset were calculated in the manner described above.
The enriched terms from the gene-based analysis are very general, hinting mainly
at roles for Stat3 in metabolic processes and regulation (\tabref{tab:supplSTAT3All}a).

\tabref{tab:supplSTAT3All}b displays the \cis-regulatory element-based enrichments 
produced by GREAT for the same set using the default settings (5+1 kb basal, up to 1 Mb extension).
%
In contrast to the generality of DAVID's term enrichments, GREAT produces many highly
specific and accurate enrichments and yields novel, testable hypotheses.
%
\emph{GO Biological Process} enrichments indicate
Stat3 regulates genes involved in both stem cell maintenance and differentiation.
%
The \emph{Mouse Phenotype} ontology shows enrichment for genes whose alteration
leads to embryonic lethality before somite formation and abnormal placental
development (\tabref{tab:supplSTAT3All}b).  Stat3 is indeed essential; Stat3 knockout mice are
embryonic lethal at early stages of development~\citep{Takeda1997}.
The \emph{PANTHER Pathway} ontology suggests that Stat3 modulates the
Interferon-$\gamma$ signaling pathway.
Interestingly, though Stat3 mediation of the Interferon-$\gamma$ pathway has
yet to be shown, Interferon-$\gamma$ has recently been shown to suppress Stat3 via
dephosphorylation~\citep{Fang2006}.
%
The \emph{MSigDB Pathway} ontology shows enrichment for genes expressed in
breast cancers, especially those involved in estrogen-receptor-dependent
signal transduction.  STAT3 expression is frequently detected in breast
cancer tissues~\citep{Hsieh2005}, though a clear link between Stat3 and
estrogen receptor expression has yet to be shown.
%
The \emph{MSigDB Perturbation}
ontology enrichment for genes upregulated after LIF treatment highlights the
link between LIF and Stat3 (\tabref{tab:supplSTAT3All}b); LIF activates the JAK/STAT signaling
pathway of which Stat3 is a part~\citep{Kisseleva2002}.
LIF is also an important factor in
uterine blastocyst implantation, though its expression is required in the utero-placental unit
rather than the blastocyst (from which ESCs are derived)~\citep{Auernhammer2000}.
Thus it is striking that GREAT highlights the \emph{Mouse Phenotype} ``abnormal trophoblast layer
morphology'' and the \emph{MGI Expression:\ Detected} enrichments for trophectoderm and extraembryonic
component in Theiler stage 5 (\tabref{tab:supplSTAT3All}b).  The binding of Stat3 to these regions
may reflect the totipotent state of ESCs or an overlap between genes expressed in trophectoderm and the
blastocyst.
The \emph{MSigDB Perturbation}
ontology also shows enrichment for genes downregulated by expression of constitutively
active JUN N-terminal kinase (JNK).  Indeed, JNK is known to be a negative regulator of Stat3~\citep{Lim1999}.
Furthermore, the strong enrichment for genes upregulated by insulin is explained by the
ability of insulin to activate Stat3~\citep{Coffer1997}.
Finally, there is enrichment for transcriptional modulators present during myeloid differentiation, and
indeed, Stat3 is an essential component for myeloid differentiation~\citep{McLemore2001}.
%

Results from a ``gene-based GREAT'' analysis recapitulate DAVID's generality in GO enrichments,
with none of the top enrichments indicating the roles of Stat3 in stem cell maintenance and differentiation
(\tabref{tab:supplStat3GBG}).  The \emph{Mouse Phenotype} ontology enrichments are also more
general than in the standard GREAT analysis, with placenta morphology identified but the 
early embryonic lethality unidentified.
%
Similarly to the p300 in embryonic stem cells example, the \emph{basal plus extension} with a
restricted enrichment domain highlights more general terms and appears more similar to the
``gene-based GREAT'' (\tabref{tab:supplStat3Basal50}), while the \emph{two nearest genes} and
\emph{single nearest gene} association rules lead to similar enrichments to the default
GREAT (\tables{tab:supplStat3Two} and \ref{tab:supplStat3One}, respectively).

Overall, by coupling appropriate integration of distal binding events with data from
many ontologies spanning a wide variety of biological phenomena, GREAT highlights
many known functions of Stat3 in mouse ESCs that gene-based
tools fail to feature prominently (\tabref{tab:supplSTAT3All}).  Experimentally-validated
links between Stat3 and its pathway involvements, its known
cofactors, and its role in stem cell maintenance are all highlighted by
various ontologies.  In addition, novel hypotheses of Stat3 involvement in
trophectoderm development and its additional cofactors can be studied by
targeted future experimentation.
%

\subsection{Neuron-restrictive silencer factor (NRSF) in human Jurkat cells}
To assess the functional roles of Neuron-Restrictive Silencer Factor (NRSF, also known as
RE1-Silencing Transcription Factor or REST), we used ChIP-Seq binding data from human
Jurkat cells~\citep{Valouev2008}.  NRSF is a transcription factor involved in
silencing neuron-specific genes~\citep{Chong1995, Schoenherr1995, Chen1998}.

A gene-based analysis of the dataset identified enrichment for
genes ``mostly involved in neuronal function''~\citep{Valouev2008}.  The top ten results
of the analysis are reproduced in \tabref{tab:supplNRSFAll}a.

We ran GREAT on a dataset comprised of the most significant ChIP-Seq peaks of
NRSF (QuEST score $> 1$; n = 1,712) using a whole genome background and
default settings.  Enriched terms overwhelmingly implicate NRSF as binding near genes involved in
ion channel activity, neurotransmitter transport, and synaptic transmission
(\tabref{tab:supplNRSFAll}b).
Additionally, the \emph{GO Cellular Component}, \emph{Mouse Phenotype}, \emph{InterPro},
and \emph{HGNC Gene Families} ontologies indicate that NRSF
binds near both calcium channel and potassium channel genes.  NRSF has been
shown to modulate aldosterone and cortisol production by regulating a calcium
channel subunit~\citep{Somekawa2009}.  NRSF has also been shown to regulate
potassium channel expression, affecting the phenotype of human vascular
smooth muscle cells~\citep{Cheong2005}.  The GREAT enrichments suggest that
NRSF may play a role in regulating other calcium and potassium channel genes
as well.
%

The enrichments are robust to all variations of association rule including a 
``gene-based GREAT'' (\tables{tab:supplNRSFGBG}--\ref{tab:supplNRSFOne}).
%
The ability of both the gene-based analyses and GREAT to 
identify the neuron-specific functions of NRSF binding data suggests that both
proximal and distal binding events play a role in the transcriptional repression of
neuron-specific genes~\citep{Chen1998}.  Given the high information
content of the 21 bp neuron-restrictive silencer element (NRSE) bound by
NRSF~\citep{Chong1995, Schoenherr1995}, binding of NRSF to the NRSE
may be predominantly functional regardless of the location of the binding area relative
to nearby genes (\figref{fig:gFig2}a).


\subsection{GA-Binding Protein (GABP) in human Jurkat cells}
The binding profile of GA-Binding Protein (GABP) was assayed in human Jurkat cells
via ChIP-Seq~\citep{Valouev2008}.  GABP is a ubiquitous transcription factor that
controls transcriptional regulation of genes involved in many diverse functions including
apoptosis, differentiation, cell cycle, and cellular energy metabolism~\citep{Rosmarin2004}.

A gene-based analysis of GABP-regulated genes showed enrichment for genes ``involved in
basic cellular processes, particularly those related to gene expression''~\citep{Valouev2008}.
The top ten results are reproduced in \tabref{tab:supplGABPAll}a.
%

Due to the large number (6,442) of GABP binding peaks present in the dataset,
more than 3,000 genes possess a GABP binding peak even within their proximal
promoter. As a result the DAVID website cannot even be used to analyze this
set, as it can only analyze datasets of 3,000 or fewer genes.
%
In contrast, GREAT can handle datasets of hundreds of thousands of genomic peaks,
and any number of resulting gene picks.  We ran GREAT on the most significant
ChIP-Seq peaks of GABP (QuEST score $> 1$, n = 3,585) using a whole genome
background and default settings (\tabref{tab:supplGABPAll}b).

Enrichments from the \emph{Pathway Commons} ontology highlight the known
functions of GABP as a transcriptional activator, as the strongest enrichment
is for genes involved in transcription.  Interestingly, there are also strong
enrichments for genes involved in various aspects of mRNA processing, with
the \emph{MSigDB Pathway} ontology highlighting genes involved in mRNA splicing
as its strongest enrichment.  GABP has been shown to regulate Hepatocyte Growth
Factor-Regulated Tyrosine Kinase Substrate, a
protein that mediates alternative mRNA splicing during liver
regeneration~\citep{Du1998,Rosmarin2004}.  The
\emph{MSigDB Pathway} and \emph{GO Molecular Function} ontologies also show
strong enrichments for ribosomal proteins.  Indeed, GABP is known to regulate
multiple ribosomal proteins~\citep{Curcic1997,Genuario1993}, though the
extent of GABP binding suggests that many other ribosomal proteins may also
be regulated by GABP.  Furthermore, GABP regulates transcription of eIF6, an
essential trans-acting factor in ribosome biogenesis~\citep{Donadini2006}.
The \emph{MSigDB Pathway} enriched terms ``oxidative phosphorylation'' and
``electron transport'' also correspond to known functions of GABP; GABP
regulates mtTFA, a mitochondrial transcription factor important in oxidative
phosphorylation~\citep{Chinenov2000}.  Finally, the \emph{Transcription
Factor Targets} ontology indicates that GABP binding peaks occur near genes
that are regulated by ETS1 and YY1, suggesting a possible cooperative role
between GABP and these factors.  GABP itself is part of the ETS family, and
ChIP-Seq experiments examining the binding of both GABP and ETS1 show that
the proteins do bind many similar promoters, though GABP is in general a more
ubiquitous factor~\citep{Collins2007}.  Interactions between GABP and YY1
have also been experimentally shown~\citep{Rosmarin2004,Delehouzee2005}.
%

As GABP binds predominantly near the promoter of its target genes
(\figref{fig:gFig2}a), the unique enrichments highlighted by GREAT
ontologies are robust to both gene-based analysis (\tabref{tab:supplGABPGBG})
and for all tested variations of association rule
(\tables{tab:supplGABPBasal50}--\ref{tab:supplGABPOne}).

\ignore{% --- begin ignore ---
\subsection{Evolutionary studies}
\label{XEvolExample}
Genome comparisons between species can identify regions that are more similar
across species than is expected under the assumption of neutral
evolution. One such set of highly constrained sequences is the
``ultraconserved elements,'' a set of 481 sequences each of length 200bp or
more, perfectly conserved between human, mouse, and rat~\citep{Bejerano2004}.

When run with the 256 non-exonic ultraconserved elements as input against a whole genome background, GREAT shows extremely
significant enrichments for proximity to genes involved in DNA binding, transcription regulation, and gene expression
regulation, many of which are homeobox genes whose knockouts are not viable (\tabref{tab:supplNonExonicUltras}).
Indeed, a large fraction of the non-exonic subset of ultraconserved elements have been shown to act as \cis-regulatory elements
for nearby developmental regulator genes~\citep{Pennacchio2006}.


Using the gene-based hypergeometric test, the 111 ultraconserved elements that overlap exons were shown to
be part of genes significantly overrepresented for RNA binding and RNA splicing functions, and the presence of the 
RNA recognition motif~\citep{Bejerano2004}.  The hypothesis that these ultraconserved elements may be involved in alternative
splicing has since been validated in multiple studies~\citep{Lareau2007, Ni2007}.
%

When run with the 111 exonic ultraconserved elements as input against a whole genome background, GREAT recapitulates the
results given in~\citep{Bejerano2004}, with the listed terms highly significant using both the binomial
and hypergeometric tests.  Terms significant at a FDR of 0.05 by both the binomial and hypergeometric test and that are
one of the top ten most significant terms in the binomial test are shown in 
\tables{tab:supplUltraWGGOBiol}--\ref{tab:supplUltraWGInterPro}.
%
To assess whether these enrichments are unique to the exonic ultraconserved elements, or are an overarching theme within
all ultraconserved elements, we ran the foreground/background hypergeometric test over genomic regions
using the 111 exonic ultraconserved elements as the foreground set and all 481 ultraconserved elements as the background set.
The same strong enrichments are seen in the foreground/background
test (\tables{tab:supplUltraBgfgGOBiol}--\ref{tab:supplUltraBgfgInterPro}), indicating that the exonic ultraconserved
elements are indeed unique within the set of all ultraconserved elements for their role in RNA processing and RNA splicing.

--- end ignore --- }

\clearpage

%------------------------------------------------------------------------------

\section{Supplementary Figures}

\begin{figure}[htbp]
\centering
\begin{tabular}{ll}
{\large {\bf a}} & {\large {\bf b}} \\
\epsfig{file=great/biasMGIExpDet.pdf,width=0.45\linewidth,clip=,trim=64 82 65 77} &
\epsfig{file=great/biasMGIExpNotDet.pdf,width=0.45\linewidth,clip=,trim=64 82 65 77} \\
{\large {\bf c}} & {\large {\bf d}} \\
\epsfig{file=great/biasMGIPheno.pdf,width=0.45\linewidth,clip=,trim=64 82 65 77} &
\epsfig{file=great/biasInterPro.pdf,width=0.45\linewidth,clip=,trim=64 82 65 77} \\
\end{tabular}
\caption[Robustness of genomic region-based and gene-based enrichment tests]{
{\bf Robustness of genomic region-based and gene-based enrichment tests.}
The gene-based hypergeometric test generates false positive enriched terms
in many ontologies when not restricted to only proximal binding events for the
{\bf (a)} \emph{MGI Expression:\ Detected},
{\bf (b)} \emph{MGI Expression:\ Not Detected},
{\bf (c)} \emph{Mouse Phenotype},
and {\bf (d)} \emph{InterPro} ontologies.
Tests were performed as described in the caption for~\figref{fig:gFig2}b.
Though the total number of average false positive terms varies across ontology (note scale changes on y-axis), the qualitative
shape of the graphs is similar with the majority of false positive enriched terms occurring for input sets containing 1-50k
elements.
}
\label{fig:gSupFig1}
\end{figure}

\begin{figure}[htbp]
\centering
\begin{tabular}{c}
\epsfig{file=great/allSkews.jpg,width=\linewidth,clip=,trim=0 3 0 0} \\
\end{tabular}
\caption[Binomial and hypergeometric \emph{P} value differences for several different datasets]{
{\bf Binomial and hypergeometric \emph{P} value differences for several different datasets.}
{\bf (a)} p300 mouse embryonic limb data set~\citep{Visel2009}.
{\bf (b)} p300 mouse embryonic forebrain data set~\citep{Visel2009}.
{\bf (c)} p300 mouse embryonic midbrain data set~\citep{Visel2009}.
{\bf (d)} NRSF human Jurkat data set~\citep{Valouev2008}.
{\bf (e)} p300 mouse embryonic stem cell data set~\citep{Chen2008}.
{\bf (f)} Stat3 mouse embryonic stem cell data set~\citep{Chen2008}.
The labeling scheme is as described in the caption for~\figref{fig:gFig2}c.
}
\label{fig:gSupFig3}
\end{figure}

\clearpage
%------------------- Supplementary Tables -----------------------------------------------------------
\begin{samepage}
\section{Supplementary Tables}
% --- SRF Tables ---
\begin{table}[htbp]
\caption[SRF ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess an SRF binding peak within 2 kb of its
transcription start site.}
}
\label{tab:supplSRFGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/SRF2kbp1.png,width=0.73\linewidth,clip=,trim=0 0 5 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\end{samepage}
\clearpage

\begin{center}
\begin{tabular}{c}
\epsfig{file=great/SRF2kbp2.png,width=0.73\linewidth,clip=,trim=0 0 0 5} \\
\end{tabular}
\end{center}


\begin{table}[htbp]
\caption[SRF 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT \cis-regulatory element enrichments of SRF using the \emph{basal plus extension} association rule with a basal
regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and a maximum extension of 50 kb,
using the highest-scoring SRF peaks anywhere in the genome (QuEST score $> 1$; n = 556).}
}
\label{tab:supplSRFBasal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/SRF50kb.png,width=0.81\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}


\begin{table}[t]
\caption[SRF two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of SRF using the \emph{two nearest genes} association rule with a maximum
extension of 1 Mb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring SRF peaks anywhere in the genome (QuEST score $> 1$; n = 556).
}
}
\label{tab:supplSRFTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/SRFTwoClosest.png,width=0.79\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[SRF single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of SRF using the \emph{single nearest gene} association rule with a maximum
extension of 1 Mb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring SRF peaks anywhere in the genome (QuEST score $> 1$; n = 556).}
}
\label{tab:supplSRFOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/SRFOneClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}


% --- P300 Tables ---
\begin{table}[htbp]
\caption[p300 limb DAVID gene-based enrichments]{
{\bf DAVID gene-based enrichments of genes with proximal p300 limb binding events.}
}
\label{tab:supplP300LimbDavid}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/limbDAVID.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[htbp]
\caption[p300 limb ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess a p300 limb binding peak within 2 kb of its
transcription start site.}
}
\label{tab:supplP300LimbGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300Limb2kb.png,width=0.75\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[htbp]
\caption[p300 limb 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT \cis-regulatory element enrichments of p300 in limb using the \emph{basal plus extension} association rule with a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and a maximum extension of 50 kb.}
}
\label{tab:supplP300LimbBasal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300Limb50kb.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[p300 limb two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 limb peaks using the \emph{two nearest genes} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300LimbTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300LimbTwoClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[p300 limb single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 limb peaks using the \emph{single nearest gene} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300LimbOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300LimbOneClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[htbp]
\caption[p300 forebrain DAVID gene-based enrichments]{
{\bf DAVID gene-based enrichments of genes with proximal p300 forebrain binding events.}
}
\label{tab:supplP300ForeDavid}
\vspace{0.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/foreDAVID.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[htbp]
\caption[p300 forebrain ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess a p300 forebrain binding peak within 2 kb of its
transcription start site.}
}
\label{tab:supplP300ForeGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300Fore2kb.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[htbp]
\caption[p300 forebrain 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT \cis-regulatory element enrichments of p300 in forebrain using the \emph{basal plus extension} association
rule with a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and a
maximum extension of 50 kb.}
}
\label{tab:supplP300ForeBasal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300Fore50kb.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[p300 forebrain two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 forebrain peaks using the \emph{two nearest genes} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300ForeTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300ForeTwoClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[p300 forebrain single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 forebrain peaks using the \emph{single nearest gene} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300ForeOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300ForeOneClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\clearpage

\begin{table}[htbp]
\caption[p300 midbrain ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess a p300 midbrain binding peak within 2 kb of its
transcription start site.}
}
\label{tab:supplP300MidGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300Mid2kb.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 midbrain 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT \cis-regulatory element enrichments of p300 in midbrain using the \emph{basal plus extension}
association rule with a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription
start site and a maximum extension of 50 kb.}
}
\label{tab:supplP300MidBasal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300Mid50kb.png,width=\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 midbrain two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 midbrain peaks using the \emph{two nearest genes} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300MidTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300MidTwoClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 midbrain single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 midbrain peaks using the \emph{single nearest gene} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300MidOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300MidOneClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 mESC comparison of DAVID and GREAT]{
{\bf GREAT enrichments for regions bound by p300 in mESCs.}
{\bf (a)} DAVID gene-based enrichments of genes with proximal p300 binding events.
{\bf (b)} GREAT \cis-regulatory element enrichments for all regions bound by p300.
}
\label{tab:supplP300ESAll}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\multicolumn{1}{l}{{\Large {\bf a}}} \\
\emph{No terms were found significant after multiple hypothesis correction}\\
\emph{in DAVID's gene-based test.} \\[15pt]
\multicolumn{1}{l}{{\Large {\bf b}}} \\
{\small {\bf GREAT Enrichments of p300 Binding Peaks in mESCs}} \\
\epsfig{file=great/p300ESbunched.png,width=0.95\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[p300 mESC ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess a p300 binding peak in mESCs within 2 kb of its
transcription start site.  Shown are the top ten hypergeometric enriched terms at a false discovery rate of 0.05.}
}
\label{tab:supplP300ESGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300ES2kbHyper.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 mESC 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT enrichments of all p300 binding peaks in mESCs using the
\emph{basal plus extension} association rule with
a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and
a maximum extension of 50 kb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300ESBasal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300ESBasal50kb.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 mESC two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 binding peaks in mESCs using the
\emph{two nearest genes} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300ESTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300ESTwoClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[p300 mESC single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all p300 binding peaks in mESCs using the
\emph{single nearest gene} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplP300ESOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/p300ESOneClosest.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\clearpage


\begin{longtable}{c}
\caption[Stat3 mESC comparison of DAVID and GREAT]{
{\bf Enrichments for regions bound by Stat3 in mESCs.} 
{\bf (a)} DAVID gene-based enrichments of genes with proximal Stat3 binding events.
{\bf (b)} GREAT \cis-regulatory element enrichments for all regions bound by Stat3.
}
\label{tab:supplSTAT3All} \\
  \\[-1.8ex]
\endfirsthead

\multicolumn{1}{c}{{\tablename} \thetable{} -- Continued} \\[0.5ex]
  \\[-1.8ex]
\endhead

  \multicolumn{1}{l}{{Continued on Next Page\ldots}} \\
\endfoot

%This is the footer for the last page of the table...
  \\[-1.8ex]
\endlastfoot
\multicolumn{1}{l}{{\Large {\bf a}}} \\
{\small {\bf DAVID Gene-based Enrichments of Stat3 Binding Peaks in mESCs}} \\
\epsfig{file=great/stat3DAVID.png,width=0.95\linewidth,clip=,trim=0 0 0 0} \\
\multicolumn{1}{l}{{\Large {\bf b}}} \\
{\small {\bf GREAT Enrichments of Stat3 Binding Peaks in mESCs}} \\
\epsfig{file=great/STAT3bunched.png,width=0.88\linewidth,clip=,trim=0 0 0 0} \\
\end{longtable}


\begin{table}[t]
\caption[Stat3 mESC ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess a Stat3 binding peak in mESCs within 2 kb of its
transcription start site.  Shown are the top ten hypergeometric enriched terms at a false discovery rate of 0.05.}
}
\label{tab:supplStat3GBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/Stat32kbHyper.png,width=0.98\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}


\begin{table}[p]
\caption[Stat3 mESC 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT enrichments of all Stat3 binding peaks in mESCs using the
\emph{basal plus extension} association rule with
a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and
a maximum extension of 50 kb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplStat3Basal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/Stat3Basal50kb.png,width=0.6\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[Stat3 mESC two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all Stat3 binding peaks in mESCs using the
\emph{two nearest genes} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplStat3Two}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/Stat3TwoClosest.png,width=0.79\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[p]
\caption[Stat3 mESC single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of all Stat3 binding peaks in mESCs using the
\emph{single nearest gene} association rule with
a maximum extension of 1 Mb. Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test.}
}
\label{tab:supplStat3One}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/Stat3OneClosest.png,width=0.75\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\clearpage


\begin{longtable}{c}
\caption[NRSF comparison of DAVID and GREAT]{
{\bf Enrichments for regions bound by NRSF in human Jurkat cells.}
{\bf (a)} The top ten proximal binding gene-based enrichments (reproduced from \citep{Valouev2008}).
{\bf (b)} GREAT \cis-regulatory element enrichments for all regions bound by NRSF.
}
\label{tab:supplNRSFAll} \\
  \\[-1.8ex]
\endfirsthead
\multicolumn{1}{c}{{\tablename} \thetable{} -- Continued} \\[0.5ex]
  \\[-1.8ex]
\endhead
  \multicolumn{1}{l}{{Continued on Next Page\ldots}} \\
\endfoot
%This is the footer for the last page of the table...
  \\[-1.8ex]
\endlastfoot
\multicolumn{1}{l}{{\Large {\bf a}}} \\
{\small {\bf Gene-based GO Enrichments of NRSF Promoter Binding Peaks}} \\
\epsfig{file=great/NRSFValouev.png,width=0.77\linewidth,clip=,trim=0 0 0 0} \\
\newpage
\multicolumn{1}{l}{{\Large {\bf b}}} \\
{\small {\bf GREAT Enrichments of NRSF Binding Peaks in Human Jurkat Cells}} \\
\epsfig{file=great/NRSFbunched.png,width=0.8\linewidth,clip=,trim=0 0 0 0} \\
\end{longtable}


\begin{table}[t]
\caption[NRSF ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess an NRSF binding peak within 2 kb of its
transcription start site.  Shown are the top ten hypergeometric enriched terms at a false discovery rate of 0.05.}
}
\label{tab:supplNRSFGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/NRSF2kbHyperAll.png,width=0.98\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\clearpage


\begin{longtable}{c}
\caption[NRSF 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT enrichments of NRSF using the \emph{basal plus extension} association rule with a maximum
a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and
extension of 50 kb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring NRSF peaks anywhere in the genome (QuEST score $> 1$; n = 1,712).}
}
\label{tab:supplNRSFBasal50} \\
  \\[-1.8ex]
\endfirsthead
\multicolumn{1}{c}{{\tablename} \thetable{} -- Continued} \\[0.5ex]
  \\[-1.8ex]
\endhead
  \multicolumn{1}{l}{{Continued on Next Page\ldots}} \\
\endfoot
%This is the footer for the last page of the table...
  \\[-1.8ex]
\endlastfoot
\epsfig{file=great/NRSFBasal50kbp1.png,width=0.85\linewidth,clip=,trim=0 0 0 0} \\
\newpage
\epsfig{file=great/NRSFBasal50kbp2.png,width=0.85\linewidth,clip=,trim=0 0 0 0} \\
\end{longtable}
\clearpage


\begin{table}[t]
\caption[NRSF two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of NRSF using the \emph{two nearest genes} association rule with a maximum
extension of 1 Mb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring NRSF peaks anywhere in the genome (QuEST score $> 1$; n = 1,712).}
}
\label{tab:supplNRSFTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/NRSFTwoClosest.png,width=0.7\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[NRSF single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of NRSF using the \emph{single nearest gene} association rule with a maximum
extension of 1 Mb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring NRSF peaks anywhere in the genome (QuEST score $> 1$; n = 1,712).}
}
\label{tab:supplNRSFOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/NRSFOneClosest.png,width=0.53\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\clearpage


\begin{longtable}{c}
\caption[GABP comparison of DAVID and GREAT]{
{\bf Enrichments for regions bound by GABP in human Jurkat cells.}
{\bf (a)} The top ten proximal binding gene-based enrichments (reproduced from \citep{Valouev2008}).
{\bf (b)} GREAT \cis-regulatory element enrichments for all regions bound by GABP.
}
\label{tab:supplGABPAll} \\
  \\[-1.8ex]
\endfirsthead
\multicolumn{1}{c}{{\tablename} \thetable{} -- Continued} \\[0.5ex]
  \\[-1.8ex]
\endhead
  \multicolumn{1}{l}{{Continued on Next Page\ldots}} \\
\endfoot
%This is the footer for the last page of the table...
  \\[-1.8ex]
\endlastfoot
\multicolumn{1}{l}{{\Large {\bf a}}} \\
{\small {\bf Gene-based GO Enrichments of GABP Promoter Binding Peaks}} \\
\epsfig{file=great/GABPValouev.png,width=0.77\linewidth,clip=,trim=0 0 0 0} \\
\newpage
\multicolumn{1}{l}{{\Large {\bf b}}} \\
{\small {\bf GREAT Enrichments of GABP Binding Peaks in Human Jurkat Cells}} \\
\epsfig{file=great/NRSFbunched.png,width=0.82\linewidth,clip=,trim=0 0 0 0} \\
\end{longtable}
\clearpage


\begin{table}[t]
\caption[GABP ``gene-based GREAT'' enrichments]{
{\bf ``Gene-based GREAT'' enrichments of all genes that possess an GABP binding peak within 2 kb of its
transcription start site.  Shown are the top ten hypergeometric enriched terms at a false discovery rate of 0.05.}
}
\label{tab:supplGABPGBG}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/GABP2kbHyperAll.png,width=0.98\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[GABP 5+1 basal up to 50 kb GREAT enrichments]{
{\bf GREAT enrichments of GABP using the \emph{basal plus extension} association rule with a maximum
a basal regulatory region extending 5 kb upstream and 1 kb downstream of the transcription start site and
extension of 50 kb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring GABP peaks anywhere in the genome (QuEST score $> 1$; n = 3,585).}
}
\label{tab:supplGABPBasal50}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/GABPBasal50kb.png,width=0.58\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}
\clearpage

\begin{table}[t]
\caption[GABP two nearest genes up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of GABP using the \emph{two nearest genes} association rule with a maximum
extension of 1 Mb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring GABP peaks anywhere in the genome (QuEST score $> 1$; n = 3,585).}
}
\label{tab:supplGABPTwo}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/GABPTwoClosest.png,width=0.7\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

\begin{table}[t]
\caption[GABP single nearest gene up to 1 Mb GREAT enrichments]{
{\bf GREAT enrichments of GABP using the \emph{single nearest gene} association rule with a maximum
extension of 1 Mb.  Shown are the top ten binomial
enriched terms at a false discovery rate of 0.05 with a fold enrichment of at least two that are also significant
by the hypergeometric test, using the highest-scoring GABP peaks anywhere in the genome (QuEST score $> 1$; n = 3,585).}
}
\label{tab:supplGABPOne}
\vspace{.1cm}
\begin{center}
\begin{tabular}{c}
\epsfig{file=great/GABPOneClosest.png,width=0.69\linewidth,clip=,trim=0 0 0 0} \\
\end{tabular}
\end{center}
\small{}
\end{table}

