\chapter{Introduction}
\label{chap:intro}

The human genome consists of 3 billion adenine, cytosine, guanine, and thymine deoxyribonucleic acid (DNA) bases that provide the instructions for making every cell in our bodies. The central dogma of biology tells us that these instructions, via ribonucleic acid (RNA) intermediates, encode the information necessary for synthesizing proteins, which in turn act as the molecular machinery of the cell. Over the past decade, the plummeting cost of DNA sequencing has allowed scientists to measure the identity of these 3 billion nucleotides, revealing that protein coding sequence only occupies ~2\% of the human genome.

The precipitous drop in the cost of sequencing has also driven scientists to dream up new methods for surveying the entirety of what is going on within the cell. In addition to measuring the identity of the 3 billion nucleotides that make up the human genome, DNA sequencing can now be used to measure variation in these nucleotides between individuals, protein binding and chemical modifications to these nucleotides, and the repertoire of genes being expressed within cells. Functional sequencing of transcription factor binding sites, epigenomic marks, open chromatin, and more, as well as the sequencing of other species and cross-species sequence conservation, suggest that as much as 10 times more genomic sequence (20\% of the genome) is dedicated to regions that control gene expression. In contrast with the amino acid code, which specifies a strict grammar for translating 3-nucleotide codons into the amino acid sequences that make up proteins, the underlying logic of these regulatory sequences remains poorly understood.

The central question of genomics is understanding the instructions encoded within our DNA - going from genotype to phenotype. From a medical perspective, the consequences of these instructions, particularly when they undergo a change due to a mutation, are of particular interest. Nature's own experiments, naturally occurring mutations resulting in disease, coupled with finding similar mutations causing the same disease in other individuals or filtering against the background of mutations found in thousands of healthy individuals, has begun to reveal the specific changes to DNA that cause disease. As the price of sequencing continues to drop, we reveal more and more of the genomic underpinnings of human diseases and also highlight sequences having biological significance.

\section{Gene regulation}

\subsection{Transcriptional regulation}

The 2\% of the human genome dedicated to protein coding sequence is divided into approximately 20,000 genes, but these are controlled by approximately 1,000,000 gene regulatory sequences. These regulatory sequences control when, where, and how much of each gene is produced in different cell types at different times~\citep{Levo:2014hl}. A typical gene regulatory enhancer is over 100 base pairs (bp) long and encodes a landing pad for multiple transcription factors to bind~\citep{Spitz:2012fj}. Through enhancer-promoter looping, these transcription factors recruit RNA polymerase II and result in the transcription of the enhancer's target gene in the appropriate spatio-temporal pattern.

\subsection{Fan in}

The mammalian genome contains multiple large gene deserts carrying numerous conserved and likely \emph{cis}-regulatory sequences~\citep{Ovcharenko:2005br}. Experiments in yeast have revealed that gene expression monotonically increases (and eventually saturates) as the number of tandem transcription factor binding sites in the promoter of that gene increases~\citep{Sharon:2012io}. The purpose of multiple distinct regulatory sequences next to the same gene that are active in the same spatio-temporal context, however, remains unknown. It is possible that multiple active enhancers in the same context buffer against environmental variability, act in an additive fashion to promote gene expression, or fine-tune the precise levels of gene expression. Additional factors provide further complications for the functional interpretation of multiple enhancers of the same gene. While cross-species conservation may reveal likely \emph{cis}-regulatory sequences, it is impossible to deduce from sequence patterns alone how many \emph{cis}-regulatory regions are active simultaneously in any given biological context.

The sequencing of the DNA bound by transcriptional co-activators or labeled by certain epigenomic marks are effective methods for annotating active enhancers in specific biological contexts genome-wide~\citep{Visel:2009jp,Ernst:2011kw}. For example, over 80\% of genomic regions marked by EP300, a transcriptional co-activator, in embryonic day 11.5 (E11.5) mouse forebrain showed enhancer activity when tested \emph{in vivo}~\citep{Visel:2009jp}. In previous work, we measured the active enhancer landscape of E14.5 dorsal cerebral cortex by harvesting tissue from mouse embryos, isolating chromatin, and performing chromatin immunoprecipitation with massively parallel DNA sequencing (ChIP-seq) for EP300. Multiple genes were flanked by dozens of candidate enhancers each, suggesting the high ``fan in" of a complex regulatory program. These heavily regulated genes include key neocortical genes, as well as suspected and novel genes~\citep{Wenger:2013jd}, suggesting that those genes with the highest ``fan in" represent important genes within a given biological context.

\subsection{Fan out}

While a single gene may have many adjacent enhancers, the converse is also true: transcription factors may regulate many different target genes. From the perspective of the regulators in the network, ChIP-seq of different transcription factors has repeatedly shown that a given transcription factor binds near hundreds or even thousands of genes specific to the contexts it is known to regulate, suggesting a high “fan out” of transcription regulation~\citep{McLean:2010iq}. This means that a given transcription factor can simultaneously activate entire pathways of downstream genes. By collectively examining the annotated functions of the target genes downstream of a given transcription factor, we can gain insights into its biological roles~\citep{McLean:2010iq, Wenger:2013ds}. Distinguishing transcription factor binding from transcription factor binding that causes a change in gene expression and an observable phenotype or loss of fitness when ablated is essential, however, for understanding the regulatory role of a transcription factor with respect to specific target genes.

\section{Identification of disease-causing mutations}

\subsection{Exome sequencing}

Protein coding DNA makes up just 2\% of the genome, but a disproportionate amount is known about these sequences relative to those comprising regulatory enhancers. Mutations resulting in changes to the amino acid sequence of a protein can have dramatic effects, which is reflected in strong cross-species sequence conservation. In contrast, due to the degeneracy of transcription factor binding motifs and regulatory redundancy, mutational changes to regulatory sequences may only result in a small phenotypic effect. By providing a smaller target, being well annotated and understood, and consisting of DNA that is generally highly intolerant to change, protein coding sequences are an attractive target for finding mutations causal for different diseases. On the other hand, protein coding sequences are generally more pleiotropic than the \emph{cis}-regulatory sequences that control them, making mutations found within them more difficult to interpret.

The sequencing of just the protein coding portions of the genome, or exome sequencing, allows researchers to probe this fraction of the genome at a much lower cost. By sequencing the exomes of families having a disease and identifying mutations that segregate with the disease, researchers can identify the causal mutation. Frequently, however, DNA samples of an entire family are not available, or a sick patient has a unique set of symptoms, of apparent genetic origin. Efforts such as The Exome Aggregation Consortium (ExAC) have aggregated exome sequencing data from 60,706 unrelated individuals, excluding individuals diagnosed with severe pediatric diseases ~\citep{Consortium:2015ib}. This resource can be used to filter out rare mutations arising in healthy individuals, providing a clearer path to the identification of the causal mutation for a disease.

\subsection{Genome-wide association studies}

Genome-wide association studies (GWAS), linkage studies, and related methods have produced an ever-growing catalog of many thousands of disease-associated mutations/SNPs. Unlike exome sequencing strategies, which require perfect or almost perfect segregation of a mutation with a disease, these methods rely on a statistical association between commonly occurring mutations and different diseases. As these methodologies allow us to agnostically scan the entire genome for mutations, we see 10 times more genomic mutations associated with diseases (90\%) that lie in the non-coding regions of the genome, and likely result not in gene body, but in gene regulatory alterations. As eleven thousand non-coding SNPs (versus less than a thousand coding SNPs) have been linked to most human genetic diseases, from developmental to psychiatric, it is critical to address these challenges and develop new approaches to elucidate the impact of non-coding SNPs on pathology and their underlying molecular mechanisms.

\section{Organization}

In this work, I explore the intersection of different features of transcriptional regulation with mutations, both protein-coding and regulatory, implicated in causing different diseases. Doing so reveals both specific developmental contexts relevant for disease pathology, as well as strategies for teasing apart the precise mechanism of action within those contexts.

Before exploring the intersection of transcriptional regulation and disease, \chapref{chap:mer130} introduces a regulatory code integrating signals from the fan out of multiple transcription factors to pattern the mammalian neocortex during embryonic development. This transcriptional code is derived from a transposable element family, MER130, which is notable because of its 73-fold enrichment in the embryonic neocortex. Outside of embryonic stem cells, no enrichment this large has ever been observed before. In addition to being more enriched than any transposable element family in any other set of EP300 bound sequences from ChIP-seq experiments in other tissues, including the forebrain at an earlier time-point, MER130 instances are specifically active within the developing neocortex. I identify a preserved code of transcriptional regulatory logic within the MER130 family and demonstrate that it is necessary for MER130’s role as an enhancer.

\chapref{chap:autism} begins with a set of high-confidence autism spectrum disorder (ASD) genes that were identified through exome sequencing studies, which have identified hundreds of genetic loci harboring variants in ASD probands but not their unaffected family members. I used ChIP-seq data, published microarray data, and \emph{in situ} hybridization to test the binding of a single transcription factor, TBR1, next to high-confidence ASD genes in the developing neocortex. TBR1 binds to many regulatory sequences adjacent to the majority of high-confidence ASD gene, reflecting a high fan out of TBR1 transcriptional regulation, as well as a high fan in of many TBR1-bound regulatory sequences next to individual ASD genes. These observations were made during cortical neurogenesis, highlighting the developing neocortex as an area of the brain relevant to ASD. Furthermore, using TBR1 fan in as a measure for prioritizing ASD genes, I identified an attractive subset of genes mutated in ASD probands that were depleted for mutations in the ExAC reference panel of healthy individuals.

\chapref{chap:zfishSnps} introduces the development of a novel screen for identifying non-coding disease-associated mutations that are deeply conserved and can be functionally tested in the model organism zebrafish, allowing for their precise molecular mechanism to be studied. The profile hidden Markov model (HMM) used to sensitively query evolutionarily distant species takes advantage of the fact that a given transcription factor can bind to multiple sequences, and these groups of closely related sequences constrain the state space evolution can occupy while preserving binding. This variation can be captured by the profile HMM and be used to find sensitive alignments. For this screen, we identified homolgous sequences in zebrafish, a vertebrate model organism with a sequenced genome, as well as an extensive genetic toolkit that enables transgenesis, mutagenesis, and knock-out experiments to study disease mechanisms. My first list of 22 regulatory sequences conserved to zebrafish includes SNPs  associated with many diverse human diseases and traits including myopia, scoliosis, nasopharyngeal carcinoma, attention deficit disorders, and restless legs syndrome. One challenge normally associated with interpreting the results from GWAS is that they report tag SNPs, which may be merely co-segregating markers with the casual mutation. By virtue of focusing on conserved sequence, I highlight GWAS SNPs that are most likely functional.

\chapref{chap:conclusion} describes the conclusions from these studies and suggests avenues for future work.